app/assets/stylesheets/stream.scss

```diff
@@ -18,6 +18,11 @@
   }
 }
 
+.stream {
+  max-height: 600px;
```

Comment:
Same here, this limits the stream to only 600px, but the stream should be as big as the window is (as it was before), this even breaks infinite scrolling in the stream.